recipe/meta.yaml

```diff
@@ -42,6 +43,12 @@ test:
     - caproto-get -h
     - caproto-get --help
     - caproto-put --help
+    - caproto-monitor --help
+    - caproto-repeater --help
+    - caproto-shark --help
+    - which caproto-defaultdict-server
+    - which caproto-spoof-beamline
+    - echo -e "\n" | caproto-spoof-beamline & (pid=$! ; sleep 5 ; kill -SIGINT $pid)
```

Comment:
This test is especially important, as it tests the wrong entry points. In case the module is in the wrong location, Python does not check it exists. The only way to verify the entry point is to run it. I did a quick test by changing the entry points to their old values and ran it again. It properly [failed](https://dev.azure.com/nsls2forge/nsls2forge/_build/results?buildId=6394&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=1541) on that test:

```py
+ echo -e '\n'
+ pid=732
+ caproto-spoof-beamline
+ sleep 5
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/caproto_1610063741464/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/bin/caproto-spoof-beamline", line 6, in <module>
    from caproto.ioc_examples.spoof_beamline import main
ModuleNotFoundError: No module named 'caproto.ioc_examples.spoof_beamline'
+ kill -SIGINT 732
```

This way we can catch the issues much faster than in [profile-collection-ci](https://github.com/NSLS-II/profile-collection-ci/blob/9a729913d22289322f3ddca72ff6aa695f4c8bf6/azure-linux.yml#L170).

Reply:
Credit: https://superuser.com/a/605887.

Reply:
@klauer, I think it will also be useful in https://github.com/conda-forge/caproto-feedstock/issues/27.

Reply:
Yeah, would be good to add it. conda-forge checks if you _have_ the entrypoints defined in setup.py mirrored in the recipe, but apparently does not check their importability 🤷

Reply:
Thanks for the feedback, @klauer. I'll sync the feedstocks soon.